data/DSLAB/projects.yaml

```
@@ -237,5 +311,13 @@ projects:
             - Failure
             - Verification
         type: Application
+        information:
+          - type: Paper
+            title: "Failure Sketching: A Technique for Automated Root Cause Diagnosis of In-Production Failures"
+            url: https://dslab.epfl.ch/pubs/gist.pdf
+            notes:
+              - label: Published in
+                text: "SOSP '15"
+                url: http://dl.acm.org/authorize?N93296
```

The others notes are using the "resolved" link
```suggestion
                url: https://dl.acm.org/doi/10.1145/2815400.2815412
```